<launch>

  <!-- Map server -->
  <node name="slam_controller_map" pkg="slam_controller" type="map_server.py">
    <param name="topic_prefix" value="/ugr/car/map" />
  </node>
  <node name="slam_controller" output="screen"  pkg="slam_controller" type="controller.py">
    <param name="stateChange" value="/ugr/car/states" />
    <remap from="/output/loopClosure" to="/ugr/car/loopclosure"/>
  </node>
  <node pkg="slam_loopclosure" output="screen" type="loopclosure" name="loopclosure">
        <remap from="/input/odometry" to="/ugr/car/odometry/filtered/fastslam" />
        <remap from="/output/loopClosure" to="/ugr/car/loopclosure"/>
        <remap from="/reset_closure" to="/ugr/srv/reset_loopclosure"/>
    </node>

</launch>